Fireworks/database.py

```diff
@@ -186,21 +186,21 @@ def __repr__(self):
 
     return SimpleTable
```

Comment:
```
class SimpleTable(Base):
        __table__ = Table(name, Base.metadata, *columns)
```
This can cause some unexpected bugs or behavior. You define Base above with declarative_base. which this SimpleTable will inherit. The problem is that it will be mapped to THAT specific base class. If you call it again it will give you warning that a similar table has already been mapped. Consider making Base an input and let the user control which base class to use.

Reply:
Base is now an input that the user can provide, and if not provided, the function creates a new declarative_base instance. The SQLalchemy warnings are gone now.

Reply:
this looks fine